python/ray/_private/runtime_env.py

```diff
@@ -113,17 +150,15 @@ def __init__(self, runtime_env_json: dict):
                 raise TypeError("runtime_env['pip'] must be of type str or "
                                 "List[str]")
 
-        if "working_dir" in runtime_env_json:
-            self._dict["working_dir"] = runtime_env_json["working_dir"]
-        else:
-            self._dict["working_dir"] = None
-
         if "uris" in runtime_env_json:
             self._dict["uris"] = runtime_env_json["uris"]
 
         if "_ray_release" in runtime_env_json:
             self._dict["_ray_release"] = runtime_env_json["_ray_release"]
 
+        if self._dict.get("pip") or self._dict.get("conda"):
+            self._dict["_ray_commit"] = ray.__commit__
```

Comment:
In the case where Ray is built from source, this is `"{{RAY_COMMIT_SHA}}"`.

Reply:
Can we do this, if runtime_env['_ray_commit'] is not set, we set it here. otherwise, use that. This should be better for test purposes.

Reply:
Sounds good, I'll make that change.